QUICK_START_GUIDE.md

```
@@ -0,0 +1,210 @@
+inv up
+
+# If no Docker, the app will use SQLite (already configured)
+```
+
+### 4. Start the Django Server
+```bash
+python manage.py runserver
+```
+
+### 5. Open Your Browser
+Navigate to: **http://localhost:8000**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)